2016-11-20 21:51:18,441 DEBUG    root            Loaded Command Group: ['gcloud', 'components']
2016-11-20 21:51:18,455 DEBUG    root            Loaded Command Group: ['gcloud', 'components', 'update']
2016-11-20 21:51:18,463 DEBUG    root            Running gcloud.components.update with Namespace(__calliope_internal_deepest_parser=ArgumentParser(prog='gcloud.components.update', usage=None, description='Ensure that the latest version of all installed components is installed on the\nlocal workstation.', version=None, formatter_class=<class 'argparse.HelpFormatter'>, conflict_handler='error', add_help=False), account=None, allow_no_backup=False, architecture_override=None, authority_selector=None, authorization_token_file=None, calliope_command=<googlecloudsdk.calliope.backend.Command object at 0xb5beaf30>, command_path=['gcloud', 'components', 'update'], component_ids=[], configuration=None, credential_file_override=None, document=None, flatten=None, format=None, h=None, help=None, http_timeout=None, log_http=None, operating_system_override=None, project=None, quiet=None, sdk_root_override=None, snapshot_url_override=None, trace_email=None, trace_log=None, trace_token=None, user_output_enabled=None, verbosity='warning', version=None).
2016-11-20 21:51:20,485 INFO     ___FILE_ONLY___ 

2016-11-20 21:51:20,487 INFO     ___FILE_ONLY___ 
Your current Cloud SDK version is: 133.0.0

2016-11-20 21:51:20,488 INFO     ___FILE_ONLY___ You will be upgraded to version: 135.0.0

2016-11-20 21:51:20,488 INFO     ___FILE_ONLY___ 

2016-11-20 21:51:20,489 DEBUG    root            (gcloud.components.update) Your current working directory is inside the Cloud SDK install root: /data/data/com.termux/files/usr/google-cloud-sdk.  In order to perform this update, run the command from outside of this directory.
Traceback (most recent call last):
  File "/data/data/com.termux/files/usr/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 740, in Execute
    resources = args.calliope_command.Run(cli=self, args=args)
  File "/data/data/com.termux/files/usr/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 1648, in Run
    resources = command_instance.Run(args)
  File "/data/data/com.termux/files/usr/google-cloud-sdk/lib/surface/components/update.py", line 99, in Run
    version=args.version)
  File "/data/data/com.termux/files/usr/google-cloud-sdk/lib/googlecloudsdk/core/updater/update_manager.py", line 752, in Update
    disable_backup = self._ShouldDoFastUpdate(allow_no_backup=allow_no_backup)
  File "/data/data/com.termux/files/usr/google-cloud-sdk/lib/googlecloudsdk/core/updater/update_manager.py", line 320, in _ShouldDoFastUpdate
    'outside of this directory.'.format(root=self.__sdk_root))
InvalidCWDError: Your current working directory is inside the Cloud SDK install root: /data/data/com.termux/files/usr/google-cloud-sdk.  In order to perform this update, run the command from outside of this directory.
2016-11-20 21:51:20,496 ERROR    root            (gcloud.components.update) Your current working directory is inside the Cloud SDK install root: /data/data/com.termux/files/usr/google-cloud-sdk.  In order to perform this update, run the command from outside of this directory.
2016-11-20 21:51:20,512 DEBUG    root            Metrics reporting process started...
